### DISCUSSION TITLE
Config.yaml file settings go back to default after container restart

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#784] [General]
it looks like this issue has cropped up again.

what i did was stop all my containers with "docker-compose stop" 
-- waited until all were fully stopped.
then manually edit several config.yaml files "target_peer_count:" settings...
then when done i start all the containers with "docker-compose up -d"... 
after a few minutes this setting changed back to the default setting for every container except for Chinilla..

no matter what value i set it to, it always goes back.. for example i tried 133 and 151 and 99 and they all go back to either 120 or 20

**Comment by guydavis** [comment]
Hi, thanks for the report.  I assume this against the [released version](https://github.com/guydavis/machinaris/wiki/Releases#release-streams) 0.8.4 of Machinaris.  Rather than modifying `target_peer_count` directly in the blockchain's config.yaml on Settings | Farming page, please try editing `SETFNTARGETPEERCOUNT` of the [Forktools](https://github.com/guydavis/machinaris/wiki/Forktools#target-peer-count) config on Settings | Tools page.   Note, if you don't want Forktools involved at all, it can be [completely disabled](https://github.com/guydavis/machinaris/wiki/Forktools#skip-forktools).

**Comment by LeroyINC** [comment]
yes 0.8.4
ahhh I see... so if that setting is available in the ForkTools.. then it will be overwritten by Forktools.

also i am assuming that "forktools_skip_build=true" would be set under each fork in the docker-compose file? if i do it that way
